Sources/KnitCodeGen/FunctionCallRegistrationParsing.swift

```diff
@@ -109,6 +113,9 @@ extension FunctionCallExprSyntax {
                 leadingTrivia: leadingTrivia,
                 functionName: .implements
             ) {
+                if forwardedRegistration.hasRedundantGetter {
+                    throw RegistrationParsingError.redundantGetter(syntax: implementsCalledMethod.calledExpression)
```

Comment:
Maybe worth adding test coverage for this too?

Reply:
Good point. Added an extra test.